kinoml/core/sequences.py

```diff
@@ -276,3 +280,75 @@ class RNASequence(Biosequence):
 
     ALPHABET = "AUCG"
     _ACCESSION_URL = "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=nuccore&id={}&rettype=fasta&retmode=text"
+
+
+class KinaseDomainAminoAcidSequence(Biosequence):
```

Comment:
I'd rather have a `KinaseSequence` object (like maybe above?) which provides a helper method to find its domains. This can rely on a private class if you want.